docs/configuration/messages.js

```diff
@@ -470,6 +470,9 @@ export default {
   // Title of the dashboard page.
   "dashboardPage.title": "My CollectionSpace",
 
+  // The tooltip for DateInput dropdowns.
+  "dateInput.tooltip": "Use the down arrow key or mouse to open the calendar",
+
```

Comment:
This file gets generated from the messages defined in src, so you don't need to update it here. I run the generateDocs.js script periodically. Eventually I'll put it in a GitHub action so it's automatic.